script.js

```diff
@@ -8,3 +8,225 @@ document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click'
+    technology: ['HTML', 'Bootstrap', 'Ruby on Rails'],
+    linkToLive: 'https://meryemsanem.github.io/desktop-version/',
+    linkToSource: 'https://github.com/meryemsanem/desktop-version',
+    image: 'icons/popup-desk.png',
+  },
+  {
+    projectName: 'Keeping track of hundreds  of components website',
+    projectDescription:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. <br><br>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.",
+    technology: ['HTML', 'Bootstrap', 'Ruby on Rails'],
+    linkToLive: 'https://meryemsanem.github.io/desktop-version/',
+    linkToSource: 'https://github.com/meryemsanem/desktop-version',
+    image: 'icons/popup-desk.png',
+  },
+  {
+    projectName: 'Keeping track of hundreds  of components website',
+    projectDescription:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. <br><br>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.",
+    technology: ['HTML', 'Bootstrap', 'Ruby on Rails'],
+    linkToLive: 'https://meryemsanem.github.io/desktop-version/',
+    linkToSource: 'https://github.com/meryemsanem/desktop-version',
+    image: 'icons/popup-desk.png',
+  },
+];
+const projectBtns = document.querySelectorAll('.see-project');
+Array.from(projectBtns).forEach((projectBtns, i) => {
+  projectBtns.addEventListener('click', () => {
+    const viewportWidth = window.innerWidth || document.documentElement.clientWidth;
+
+    if (viewportWidth >= 768.1) {
+      const popDiv = document.createElement('div');
+      popDiv.id = 'background-popup';
+      popDiv.style.position = 'fixed';
+      popDiv.style.display = 'block';
+      popDiv.style.top = '112px';
+      popDiv.style.left = '142px';
+      popDiv.style.width = '80vw';
+      popDiv.style.height = '100vh';
+      popDiv.style.zIndex = '11';
+      popDiv.innerHTML = `
+      <div class="popup-container">
+        <div class='desk-pop'>
+            <button id='close' class='close'>X</button>
+          </div> <br><br>
+              <h3 id='name'>${projectsArr[i].projectName}</h3> <br>
+     <div class='card-det'>
+              <ul class='skill'>
```

Comment:
- [ ] You did great having the pop-up generated dynamically with Js, :+1: But I think there are some improvements you need to make here.
- - Your pop-up for the mobile and desktop doesn't look like the given design, Your popup does not have a background color and also the texts are not aligned properly, I'll suggest you look at the given design and update your styles. :smile: :small_airplane: 

- Your current desktop design:
![Screenshot from 2023-05-24 15-54-11](https://github.com/meryemsanem/desktop-version/assets/107632780/e7669078-982b-48e9-af47-c7564f7521a7)

- Figma desktop design:
![Screenshot from 2023-05-24 15-54-35](https://github.com/meryemsanem/desktop-version/assets/107632780/7e7bfcc9-767a-436a-bf1e-b2e2e013ecda)

- Your current mobile design:
![Screenshot from 2023-05-24 15-55-46](https://github.com/meryemsanem/desktop-version/assets/107632780/5348e34f-4b8c-47d9-a443-6aca901a3663)

- Figma mobile design:
![Screenshot from 2023-05-24 15-56-03](https://github.com/meryemsanem/desktop-version/assets/107632780/bf45f7a7-0cd5-4ed5-bbab-9d9d134373f5)